debian/control.in

```diff
@@ -70,6 +71,7 @@ Depends: ${misc:Depends},
          gnome-settings-daemon (>= 3.25.90),
          gsettings-desktop-schemas (>= 3.21.4),
          udisks2 (>= 2.1.8),
+         libmogwai-tariff-0,
```

Comment:
Same.

Why do we have both `debian/control.in` and `debian/control` committed to git? Presumably `debian/control` is generated from the `.in` file, so shouldn’t need to be in git? Or does Jenkins need it?

Reply:
I believe Jenkins need it. I've never questioned that myself though...